examples/lusid/returns/1-create-transaction-portfolio.sql

```diff
@@ -0,0 +1,30 @@
+-- ============================================================
+-- Description:
+-- 1. In this query, we create a Transaction Portfolio in LUSID
```

Comment:
Can you remove the numbering as there is only 1 point?

Reply:
Removed.